base/bitset.jl

```diff
@@ -247,20 +247,7 @@ function _matched_map!(f, a1::Bits, b1::Int, a2::Bits, b2::Int,
     b1 # the new offset
 end
 
-
-@noinline _throw_bitset_bounds_err() =
-    throw(ArgumentError("elements of BitSet must be between typemin(Int) and typemax(Int)"))
-
-@inline _is_convertible_Int(n) = typemin(Int) <= n <= typemax(Int)
```

Comment:
CI points out that someone else is using `_is_convertible_Int`.

Reply:
I've added unit tests for this so we don't have to rely on unrelated Dates.jl tests to spot this type of error.